pkgs/top-level/python-packages.nix

```
@@ -1878,6 +1878,8 @@ in {
 
   kafka-python = callPackage ../development/python-modules/kafka-python {};
 
+  confuse = callPackage ../development/python-modules/confuse/default.nix {};
```

defaults to default.nix, just need the directory
```suggestion
  confuse = callPackage ../development/python-modules/confuse {};
```